static/templates/input_pill.handlebars

```diff
@@ -2,8 +2,8 @@
     {{#if has_image}}
     <img class="pill-image" src={{img_src}} />
     {{/if}}
-    {{ display_value }}
-    <div class='exit'>
-        &times;
+    <span class="pill-value">{{ display_value }}</span>
+    <div class="exit" aria-label="{{t Remove pill }}">
```

Comment:
The `aria-label` here is invalid -- the string should be quoted.  But I also don't think we really need the `aria-label` here; this isn't a widget you would want to interact with as a blind user (instead, you just hit backspace to get rid of it, I think?).